src/error.rs

```diff
@@ -6,6 +6,34 @@ use proc_macro2::Span;
 pub struct Error;
 
 impl Error {
+	/// `derive_where` was already applied on this item before.
+	pub fn visited(span: Span) -> syn::Error {
+		syn::Error::new(
+			span,
+			"`#[derive_where(..)` was already applied to this item before, this occurs when using \
+			 a qualified path for any `#[derive_where(..)`s except the first",
+		)
+	}
```

Comment:
Is there a UI test for this? or did I just overlook it?

Reply:
No, I somehow missed that.

Reply:
did you add one? because this was resolved but I could not find it?

Reply:
https://github.com/ModProg/derive-where/pull/37/files#diff-b4e400fe186fc9514f7d7023c38062fcb10c451c6f5111a7f241f646e1d8f390R83-R89
here

Reply:
oh ok, nvm